docs/_pages/releases.md

```diff
@@ -50,6 +50,7 @@ sidebar:
 * One overload of the `AssertionScope` constructor would not create an actual scope associated with the thread - [#2607](https://github.com/fluentassertions/fluentassertions/pull/2607)
 * Fixed `ThrowWithinAsync` not respecting `OperationCanceledException` - [#2614](https://github.com/fluentassertions/fluentassertions/pull/2614)
 * Fixed using `BeEquivalentTo` with an `IEqualityComparer` targeting nullable types - [#2648](https://github.com/fluentassertions/fluentassertions/pull/2648)
+* Fixed `ArgumentOutOfRangeException` when using globally enabled tracing - [#2721](https://github.com/fluentassertions/fluentassertions/pull/2721)
```

Comment:
The breaking change (`StringBuilderTraceWriter` => `StringBuilderTraceWriterPool`) should be mentioned!?

Reply:
Hmm.. Or leave the original visibility?